recipes/inferno/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.1.8" %}
+# {% set checksum = "c644f3457b31be2fa0ce8fce15536e2eaeab160d" %}
+
+
+package:
+    name: inferno
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/inferno-pytorch/inferno
+    git_tag: master
+    # TODO checkout url instead
+    # url: https://github.com/constantinpape/z5/archive/{{ version }}.tar.gz
+    # sha256: {{ checksum }}
+
+
+build:
+    number: 0
+    skip: True  # [not py36]
```

Comment:
Please, remove this selector and in build and also in run requirements do something like
```
- python <3.6
```

Reply:
For now we can only build this package on linux, because of the `torchvision` and `pytorch`
dependencies.
Can I still switch to `noarch` in this setting, or do I need to add 
`skip: True [not linux]`
to the build section?

Reply:
Oh, that's ok then